sentences/sv/_common.yaml

```diff
@@ -49,6 +49,7 @@ lists:
 expansion_rules:
   name: "{name}"
   area: "{area}"
+  område: "<area>[(et | en | s | met)]"
```

Comment:
Instead of adding a new expansion rule, why not add it to `area` one? It exists exactly for the reason that one might want to add optional things around it.

Reply:
ok, will redo it with the area instead, did get the feeling that I should translate it to Swedish also for the expansion_rules from previous PR. And then removing the old area if we don't need it anymore.